src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyNameFormatter.cs

```diff
@@ -15,7 +15,7 @@ public static string ComputeDisplayName(string? name, Version? version, string?
         {
             const int PUBLIC_KEY_TOKEN_LEN = 8;
 
-            if (name == string.Empty)
+            if (name.Length == 0)
```

Comment:
The build is complaining here because name might be null.
The null case can never happen because the [caller](https://github.com/dotnet/runtime/blob/91c1d7cab87653fc2214d9effbee54d3d9072c65/src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyName.cs#L218) checks it before passing the argument.
Should I change this line to `if (string.IsNullOrEmpty)` ?